science/veryfasttree/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        citius veryfasttree 4.0.3 v
+github.tarball_from archive
+revision            0
+
+categories          science
+license             GPL-3
+platforms           darwin
+maintainers         usc.es:cesaralfredo.pineiro
+description         Efficient phylogenetic tree inference for massive taxonomic datasets
+long_description    {*}${description}
+homepage            https://github.com/citiususc/veryfasttree
+
+checksums           rmd160  a4070bf0ddc4aa0448f9a7878a3587a9250197a0 \
+                    sha256  770e41773806c65e60ebc77363beccf40e3d339affd97c07eb7b2e1acb51af60 \
+                    size    898112
+
+depends_lib         port:libomp
+
+configure.args-append -DUSE_NATIVE=OFF
+configure.cxxflags-append "-I${prefix}/include/libomp -L${prefix}/lib/libomp"
```

Comment:
to get the `OpenMP` part to work you might want to check the "CompilerSelection" entry in the [wiki](https://trac.macports.org/wiki/CompilerSelection).
I *think* adding `compiler.openmp_version` with the correct value should probably be all that is needed as the dependency of `libomp` and adding of flags would happen automatically, but please check.

Reply:
I used "compiler.openmp_version 2.5" but I had to add "configure.cxxflags-append -fopenmp" to make it work. The find_package(OpenMP) of cmake seems not to do it automatically in this case. I hope this workaround is acceptable.